avocado/core/jobdata.py

```diff
@@ -21,7 +21,9 @@
 import json
 import os
 import pickle
+import sys
 
+from . import jobdata_compat_36_to_52
```

Comment:
This commit is pretty much unique, in the sense that makes sense for 52.0 only. While it has makes use of import magic that is usually best not to be invoked, it's very specific, so it looks OK to me.